### ISSUE TITLE
The problem is the next: the following code appear to be ok, but the page has a problem.... it clones the #target5 element and append it as well but it do it twice in the practice page.

### ISSUE THREAD

**Issue by drakedey** [freeCodeCamp/freeCodeCamp#14468]
![captura](https://cloud.githubusercontent.com/assets/16444920/25110029/21224f34-23af-11e7-83e5-d1ef668038d2.PNG)
Challenge [Target the Children of an Element Using jQuery](https://www.freecodecamp.com/challenges/target-the-children-of-an-element-using-jquery) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.133 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```html

<script>
  $(document).ready(function() {
    $("#target1").css("color", "red");
    $("#target1").prop("disabled", true);
    $("#target4").remove();
    $("#target2").appendTo("#right-well");
    $("#target5").clone().appendTo("#left-well");
    $("#target1").parent().css("background-color", "red");

  });
</script>

<!-- Only change code above this line. -->

<div class="container-fluid">
  <h3 class="text-primary text-center">jQuery Playground</h3>
  <div class="row">
    <div class="col-xs-6">
      <h4>#left-well</h4>
      <div class="well" id="left-well">
        <button class="btn btn-default target" id="target1">#target1</button>
        <button class="btn btn-default target" id="target2">#target2</button>
        <button class="btn btn-default target" id="target3">#target3</button>
      </div>
    </div>
    <div class="col-xs-6">
      <h4>#right-well</h4>
      <div class="well" id="right-well">
        <button class="btn btn-default target" id="target4">#target4</button>
        <button class="btn btn-default target" id="target5">#target5</button>
        <button class="btn btn-default target" id="target6">#target6</button>
      </div>
    </div>
  </div>
</div>

```

**Comment by williampmills** [comment]
The reason it happens is because the first instance is generated by the live updating of the preview as you code, and the second is triggered by you pressing the submit button.

It's an artifact of the way the pages are set up.

**Comment by dtm111** [comment]
this is a duplicate of #5072 and is addressed in #7430
